skiko/src/commonMain/kotlin/org/jetbrains/skia/shaper/TextBlobBuilderRunHandler.kt

```diff
@@ -73,6 +73,7 @@ class TextBlobBuilderRunHandler<T> internal constructor(
     }
 
     init {
+        // TODO [pavel.sergeev] what `manageText` == false is actually mean?
```

Comment:
I can only guess, but probably it was implemented before String has RefCnt. And now we can use acquire/release instead somehow (but I am not sure)